images/linux/scripts/installers/firefox.sh

```diff
@@ -23,3 +23,26 @@ echo "Lastly, documenting what we added to the metadata file"
 #           ($HOME is /home/packer which is owned by packer.)
 HOME=/root
 DocumentInstalledItem "Firefox ($(firefox --version))"
+
+# Download and unpack latest release of geckodriver
+URL=https:`curl -s https://api.github.com/repos/mozilla/geckodriver/releases/latest|grep 'browser_download_url.*linux64.tar.gz'|sed -e 's/^.*"https://'|sed -e 's/"//g'`
+echo "Downloading geckodriver $URL..."
+wget "$URL" -O geckodriver.tar.gz
+tar -xzf geckodriver.tar.gz
+rm geckodriver.tar.gz
+
+GECKODRIVER_BIN="/usr/bin/geckodriver"
+mv "geckodriver" $GECKODRIVER_BIN
+chown root:root $GECKODRIVER_BIN
+chmod +x $GECKODRIVER_BIN
+echo "GECKOWEBDRIVER=$GECKODRIVER_BIN" | tee -a /etc/environment
```

Comment:
Could you please take a look at this PR? Could you please install geckodriver using consistent way.
https://github.com/actions/virtual-environments/pull/360/files

I mean that we should create the separate folder for driver, something like `/usr/local/share/firefox_driver` or `/usr/local/share/gecko_driver`, put `geckodriver` there. and set `GeckoWebDriver` variable to point to the folder instead fo binary file. Also we need to make symlink to "/usr/bin/geckodriver".

We do that to be consistent with guide: https://docs.microsoft.com/en-us/azure/devops/pipelines/test/continuous-test-selenium?view=azure-devops#decide-how-you-will-deploy-and-test-your-app